eml/screens/lectures/CompSwipeScreen.jsx

```diff
@@ -48,7 +48,7 @@ const CompSwipeScreen = ({ route }) => {
 		const fetchData = async () => {
 			try {
 				const studentInfo = await getStudentInfo();
-				let initialIndex = findIndexOfUncompletedComp(studentInfo, parsedCourse.courseId, section.sectionId);
+				let initialIndex = parsedComponentIndex ? parsedComponentIndex : findIndexOfUncompletedComp(studentInfo, parsedCourse.courseId, section.sectionId);
```

Comment:
should change this to utilize the nullish coalescing operator, it's better practice in react (returns left side if not null or undefined and right side if it is).  "const initialIndex = parsedComponentIndex ?? findIndexOfUncompletedComp(studentInfo, parsedCourse.courseId, section.sectionId);"